addons/movments_accounts_balances/controllers/controllers.py

```diff
@@ -97,82 +95,95 @@ def general_ledger_report(self, account_id, start_date, end_date):
             'ledger_data': ledger_data,
         }
 
+    ##create/get/delete_bills
+    @api.model
+    def create_bill(self, bill_vals, partner_id, bill_date, bill_date_due, reference, narration):
+        """
+        Create a bill and corresponding analytic lines based on the provided data.
+        """
+        # Prepare the invoice lines
+        bill_lines = []
+        for line in bill_vals:
+            # Create an analytic line if the analytic account ID is provided
+            analytic_line_vals = []
+            if 'analytic_account_id' in line:
+                analytic_line_vals.append((0, 0, {
+                    'account_id': line['analytic_account_id'],
+                    'name': line.get('description', ''),
+                    # Include additional fields for the analytic line here as needed
+                }))
```

Comment:
@sameerz09 we need analytic_account_id (acting like QBO Class) and account_id (acting like QBO chart of account) at the same level. Please refer to this code snippet in my PR:
<img width="754" alt="image" src="https://github.com/sugampandey/odoo/assets/1596925/db48e768-78a6-4b97-a278-da0f3d2e8a78">

https://github.com/sugampandey/odoo/pull/12/files